### PR TITLE
Fixed tests + updated version to 6.4.1

### PR DESCRIPTION
**Pull request by ekkinox**
Fixed tests + updated version to 6.4.1

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -35,7 +35,7 @@
     'label' => 'Item core extension',
     'description' => 'TAO Items extension',
     'license' => 'GPL-2.0',
-    'version' => '6.4.0',
+    'version' => '6.4.1',
     'author' => 'Open Assessment Technologies, CRP Henri Tudor',
     'requires' => array(
         'taoBackOffice' => '>=3.0.0',
```
Comment (alexeyvy) [inline manifest.php]: It has to require at least generis 8.0.0 which introduced GenerisTestCase class

Comment (ekkinox) [reply manifest.php]: fixed, thx :)

File: test/integration/ItemsTest.php [hunk 2]
```diff
@@ -184,92 +194,77 @@ public function testIsItemClass()
...
     {
-        $item = $this->prophesize('core_kernel_classes_Resource');
-        $itemModel = $this->prophesize('core_kernel_classes_Resource');
-        $itemModel->getOnePropertyValue(new core_kernel_classes_Property(taoItems_models_classes_itemModel::CLASS_URI_RUNTIME))
-            ->willReturn('returnValue');
-        $item->getOnePropertyValue(new core_kernel_classes_Property(taoItems_models_classes_ItemsService::PROPERTY_ITEM_MODEL))
-            ->willReturn($itemModel->reveal());
-        
-        $this->assertEquals('returnValue', $this->itemsService->getModelRuntime($item->reveal()));
+        $item = $this->ontologyMock->getResource(taoItems_models_classes_ItemsService::PROPERTY_ITEM_MODEL);
```
Comment (siwane) [inline test/integration/ItemsTest.php]: It is confusing to have same uri for two different resources, also `new core_kernel_classes_resource` can be replaced `$this->ontologyMock->getResource`

File: test/integration/ItemsTest.php [hunk 3]
```diff
@@ -184,92 +194,77 @@ public function testIsItemClass()
...
-        $item = $this->prophesize('core_kernel_classes_Resource');
-        $itemModelProphecy = $this->prophesize('core_kernel_classes_Resource');
-        $itemModel = $itemModelProphecy->reveal();
-        $item->getOnePropertyValue(new core_kernel_classes_Property(taoItems_models_classes_ItemsService::PROPERTY_ITEM_MODEL))
-        ->willReturn($itemModel);
-        $this->assertEquals($itemModel, $this->itemsService->getItemModel($item->reveal()));
+        $item = $this->ontologyMock->getResource('item');
+        $this->assertNull($this->itemsService->getItemModel($item));
+
+        $model = new core_kernel_classes_Resource(taoItems_models_classes_ItemsService::PROPERTY_ITEM_MODEL);
```
Comment (siwane) [inline test/integration/ItemsTest.php]: Same here avoid to use resource constructor please

**Pull request merged**